src/components/layout/header/UserMenu/UserMenu.module.scss

```
@@ -0,0 +1,65 @@
+.container {
+  @include column-flexbox(center, center, 1.6rem);
```

```suggestion
  @include column-flexbox($gap: 1.6rem);
```
flexbox mixin의 기본속성이 center로 되어있어서 위에 처럼만 쓰셔도 됩니다.

---

반영했습니다!